server/prisma/migrations/20240527134914_switch_to_uuids/migration.sql

```diff
@@ -0,0 +1,793 @@
+ALTER TABLE "_BruteToTournament" DROP COLUMN "B";
+ALTER TABLE "_BruteToTournament" ADD COLUMN "B" uuid NOT NULL ;
+
+-- AlterTable
+ALTER TABLE "_Opponents" ADD COLUMN "temp_A" INT;
+UPDATE "_Opponents" SET "temp_A" = "A";
+ALTER TABLE "_Opponents" DROP COLUMN "A";
+ALTER TABLE "_Opponents" ADD COLUMN "A" uuid NOT NULL ;
+ALTER TABLE "_Opponents" ADD COLUMN "temp_B" INT;
+UPDATE "_Opponents" SET "temp_B" = "B";
+ALTER TABLE "_Opponents" DROP COLUMN "B";
+ALTER TABLE "_Opponents" ADD COLUMN "B" uuid NOT NULL ;
+
+
+-- UPDATE DATAS
+UPDATE "Achievement" SET "bruteId" = "Brute"."id" FROM "Brute" WHERE "Achievement"."temp_bruteId" = "Brute"."temp_id";
+UPDATE "Brute" SET "masterId" = "id" WHERE "temp_masterId" = "temp_id";
+UPDATE "Brute" b SET "clanId" = c."id" FROM "Clan" c WHERE b."temp_clanId" = c."temp_id" AND b."temp_id" IS NOT NULL;
+UPDATE "BruteInventoryItem" bii SET "bruteId" = b."id" FROM "Brute" b WHERE bii."temp_bruteId" = b."temp_id" AND bii."temp_bruteId" IS NOT NULL;
+UPDATE "BruteReport" br SET "bruteId" = b."id" FROM "Brute" b WHERE br."temp_bruteId" = b."temp_id" AND br."temp_bruteId" IS NOT NULL;
+UPDATE "Clan" c SET "masterId" = b."id" FROM  "Brute" b WHERE c."temp_masterId" = b."temp_id" AND c."temp_masterId" IS NOT NULL;
+
+UPDATE "ClanPost" cp
+SET "authorId" = b."id",
+    "threadId" = ct."id"
+FROM "Brute" b, "ClanThread" ct
+WHERE b."temp_id" = cp."temp_authorId"
+  AND ct."temp_id" = cp."temp_threadId"
+  AND cp."temp_authorId" IS NOT NULL
+  AND cp."temp_threadId" IS NOT NULL;
+
+UPDATE "ClanThread" ct
+SET "clanId" = c."id" , "creatorId" = b."id"
+FROM "Clan" c ,"Brute" b
+WHERE c."temp_id" = ct."temp_clanId"
+    AND b."temp_id" = ct."temp_creatorId"
+    AND ct."temp_clanId"  IS NOT NULL
+    AND ct."temp_creatorId" IS NOT NULL;
+
+
+UPDATE "BossDamage" bd
+SET "clanId" = c."id" , "bruteId" = b."id"
+FROM "Clan" c , "Brute" b
+WHERE c."temp_id" = bd."temp_clanId" AND b."temp_id" = bd."temp_bruteId" AND bd."temp_clanId"  IS NOT NULL AND bd."temp_bruteId" IS NOT NULL;
+
+
+UPDATE "BruteStartingStats" bss
+SET  "bruteId" = b."id"
+FROM  "Brute" b
+WHERE b."temp_id" = bss."temp_bruteId" AND bss."temp_bruteId" IS NOT NULL;
+
+UPDATE "DestinyChoice" dc SET "bruteId" = b."id" FROM "Brute" b WHERE b."temp_id" = dc."temp_bruteId"  AND dc."temp_bruteId" IS NOT NULL;
+
+UPDATE "Fight" f
+SET "brute1Id" = b1."id" ,  "brute2Id" = b2."id" , "tournamentId" = t."id"
+FROM "Brute" b1, "Brute" b2, "Tournament" t
+WHERE b1."temp_id" = f."temp_brute1Id" AND b2."temp_id" = f."temp_brute2Id" AND t."temp_id" =  f."temp_tournamentId"
+    AND f."brute1Id"  IS NOT NULL
+    AND f."brute2Id"  IS NOT NULL
+    AND f."tournamentId"  IS NOT NULL;
+
+UPDATE "Log" l
+SET "currentBruteId" = b."id" , "fightId" = f."id"
+FROM "Brute" b , "Fight" f
+WHERE b."temp_id" = l."temp_currentBruteId"
+    AND f."temp_id" = l."temp_fightId"
+    AND l."currentBruteId" IS NOT NULL
+    AND l."fightId" IS NOT NULL;
```

Comment:
`l."currentBruteId"` can't be null.
Fight should not be updated as logs with a fightId should have been deleted